languages/csharp/exercises/concept/inheritance/InheritanceTests.cs

```diff
@@ -0,0 +1,207 @@
+using Xunit;
+
+public class RolePlayingGameTests
+{
+    [Fact]
+    public void CreateNewWizard()
+    {
+        var wizard = new Wizard();
+        Assert.NotNull(wizard);
+    }
+
+    [Fact(Skip = "Remove this Skip property to run this test")]
+    public void CreateNewWarrior()
+    {
+        var warrior = new Warrior();
+        Assert.NotNull(warrior);
+    }
+
+    [Fact(Skip = "Remove this Skip property to run this test")]
+    public void DisplayHitpointsOfNewWizard()
```

Comment:
For consistency hit points was being used as two words above, so it should be here in the tests as well (applies to the next test as well)

Reply:
Done! :)